packages/colors/package.json

```diff
@@ -3,9 +3,16 @@
     "version": "3.0.0",
     "description": "Blueprint color definitions",
     "main": "lib/index.js",
+    "style": "lib/css/colors.css",
+    "sideEffects": [
+        "**/*.css"
+    ],
     "scripts": {
         "clean": "rm -rf lib/*",
-        "compile": "tsc -p src/",
+        "compile": "run-p \"compile:*\"",
+        "compile:esm": "tsc -p src/",
+        "compile:css": "sass-compile ./src",
```

Comment:
we _do_ still want this so that we can validate that we're using valid Sass syntax.